ckanext/opendatani/templates/report/view.html

```diff
@@ -0,0 +1,26 @@
+{% extends "page.html" %}
+
+{% block title %}{{ 'Publisher Report' }} - {{ _('Reports') }} - {{ super() }}{% endblock %}
+
+
+{% block breadcrumb_content %}
+  {{ h.build_nav('home', _('Home')) }}
+{% endblock%}
+
+{% for csv_id in extra_vars %}
+  {% block primary_content_inner %}
+      <h1>{{ 'Publisher Report' }}</h1>
+      <p>{{ 'An up to date report instantly generated with publisher usage data.' }}</p>
+      <div class="panel panel-info">  
+          <div class="panel-body">
+            <form action="" method="POST">
```

Comment:
`POST`? Sound more like `GET` to me

Reply:
After some testing, I actually don't need that form for my functionality. It was left over from an earlier iteration.